generator/generator_tool/generator_tool.cpp

```diff
@@ -374,6 +383,17 @@ int main(int argc, char ** argv)
       }
     }
 
+    if (FLAGS_generate_regions_kv)
+    {
+      CHECK(FLAGS_generate_region_features, ("Option --generate_regions_kv use only with "
```

Comment:
s/use/can be used/
s/with/together with/

Reply:
++